tests/compte/creerCompteUtilisateur.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { CreerCompteUtilisateurUsecase } from "../../src/compte/creerCompteUtilisateur.usecase";
+import { SessionRepository } from "../../src/authentification/authentifierUtilisateur.usecase";
+import { Utilisateur } from "../../src/authentification/ports/utilisateur.repository";
+import { CompteUtilisateur, CompteUtilisateurRepository } from "../../src/compte/ports/compteUtilisateur.repository";
+
+class SessionRepositoryForTest implements SessionRepository {
+  get utilisateur(): Utilisateur {
+    return this._utilisateur;
+  }
+  private _utilisateur: Utilisateur = {
+    id: "",
+    nom: "",
+  };
+  sauvegarderUtilisateur(utilisateur: Utilisateur) {
+    this._utilisateur = utilisateur;
+  }
+}
+
+class CompteUtilisateurForTest implements CompteUtilisateurRepository {
+  creerCompteUtilisateur(nom: string, email: string): Promise<CompteUtilisateur> {
+    return Promise.resolve({
+      id: "id",
+      nom: nom,
+      mail: email,
+      codePostal: "77650",
+    });
+  }
+
+  getCompteUtilisateur(idUtilisateur: string): Promise<CompteUtilisateur> {
+    throw Error;
+  }
+
+  mettreAjour(compteUtilisateur: CompteUtilisateur) {}
+}
+describe("Fichier de tests concernant la creation du compte utilisateur", () => {
```

Comment:
J'aurai tendance à importer `describe`, `it`, ... de vite pour avoir le bon typage

Reply:
Pour ces tests là, je ne pense pas car ce sont pas des tests vitest mais jest (clean archi)
En revanche pour les tests UI c'est à faire.